lib/tasks/locale.rake

```diff
@@ -119,4 +119,15 @@ namespace :locale do
       store['human_locale_names'] = locale_hash
     end
   end
+
+  desc "Extract model attribute names and virtual column names"
+  task "store_model_attributes" do
+    require 'gettext_i18n_rails/model_attributes_finder'
+    require_relative 'model_attribute_override.rb'
+
+    attributes_file = 'config/locales/model_attributes.rb'
+    File.unlink(attributes_file) if File.exist?(attributes_file)
+
```

Comment:
As things are, we have the `model_attributes.rb` file saved as `config/model_attributes.rb`, not
`config/locales/model_attributes.rb`.

If we leave the file in `config/locales/model_attributes.rb`, our application won't even start, because fast gettext
will think it's a gettext catalog of some sort.

So what we need to do here is to move `config/locales/model_attributes.rb` to `config/model_attributes.rb`
after the task finishes.